workflow/usage.rst

```diff
@@ -80,7 +81,8 @@ like this:
                 <framework:workflow name="blog_publishing" type="workflow">
                     <framework:audit-trail enabled="true"/>
 
-                    <framework:marking-store type="single_state">
+                    <framework:marking-store type="method">
+                      <framework:argument>false</framework:argument>
                       <framework:argument>currentPlace</framework:argument>
                     </framework:marking-store>
 
```

Comment:
I don't use XML config, can someone please provide the `initial_places` code for it.

Reply:
You can add many like this https://github.com/symfony/symfony/pull/30468/files#diff-0b1fae456d162214b077b0e1be47b40cR11.

Reply:
It would be nice to add it in the code doc.
[Here](https://github.com/symfony/symfony/blob/0d365a8c7fa0ef8822625d7ff4b5c725dfd07169/src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/workflows.xml#L11) is the final ref

Reply:
I believe I've done this correctly!